fe/fe-core/src/main/java/com/starrocks/qe/StmtExecutor.java

```diff
@@ -1700,7 +1701,7 @@ public void handleDMLStmt(ExecPlan execPlan, DmlStmt stmt) throws Exception {
             QeProcessorImpl.INSTANCE.registerQuery(context.getExecutionId(), queryInfo);
             coord.exec();
             coord.setTopProfileSupplier(this::buildTopLevelProfile);
-            coord.setExecPlanSupplier(() -> execPlan);
```

Comment:
why change the functional interface? you can keep it, when get() is called, i think i will return the new one